### ISSUE TITLE
Can't delete folder

### ISSUE THREAD

**Issue by unik-web** [nextcloud-snap/nextcloud-snap#631]
Hi,

Can't delete folder, I was have tried this solution: [https://help.nextcloud.com/t/file-is-locked-how-to-unlock/1883](https://help.nextcloud.com/t/file-is-locked-how-to-unlock/1883). I didn't found no locks in my db. 

```
Fatal | webdav | OCA\DAV\Connector\Sabre\Exception\FileLocked: "Photos/xxxxxx" is locked | 2018-07-09T14:34:19+0200
-- | -- | -- | --
Error | no app in context | OCP\Lock\LockedException: "files/a6165ed12f57e32a94bbc145a91c18b8" is locked, existing lock on file: -3924 shared locks
```
Thanks

**Comment by rviertel** [comment]
I'm having the same issue, I can't figure out how to open the database in MariaDB since I installed nextcloud as a snap

**Comment by unik-web** [comment]
Hi,
Nice to hear james, my issue was able to self remove after waiting couple of hours. So my issue is fixing by it self with just waiting. So I hope in further they comming with some updates that is fixing this.

**Comment by rviertel** [comment]
Good to know. I am using the desktop sync client and the lock occurred when
I copied a directory over and realized I put it in the wrong place, after
trying to move it things locked up. After 2 days the lock cleared and I was
able to delete the extraneous directory. Thanks for the info on the
database command.

-- Ryan Viertel

**Comment by kyrofa** [comment]
Can anyone confirm that this is fixed in the `stable` channel (13.0.5) these days?
